src/components/VDatePicker/VDatePicker.vue

```diff
@@ -72,6 +73,7 @@
     },
 
     computed: {
+      supportsLocaleFormat: () => supportsLocaleFormat,
```

Comment:
Does this need to be reactive? I don't think the browser's javascript support is likely to change at runtime